doc/integrator/create_application.rst

```diff
@@ -226,6 +226,80 @@ Commit and push on the main repository:
     git commit -m "Initial commit"
     git push origin master
 
+Configuration of different environment in your project
+------------------------------------------------------
+
+Concepts
+........
+
+* Makefile: These files are environment configuration files. Each environment will
+  have its configuration file (developer, preprod, prod)
+* vars_xxx.yaml: These files are application configuration files. Generally only
+  one file is needed to configure your application.
+
+Hierarchy and extending your configuration files
+................................................
+
+The configuration files (Makefile and vars) have a hierarchy between them.
+These files extend other files. A Make file extends another Make file
+and similarly a vars file extends another vars file. This extension is visible
+in Makefile files::
+
+    extends: CONST_vars.yaml
+
+and vars files::
+
+    extends: CONST_vars.yaml
+
+CONST files are files that should not be changed because they are replaced
+during application updates, so your changes will be systematically lost. You can
+extend these files as many times as you like, although it is not recommended to
+exceed 3-4 levels for readability and simplicity.
+
+.. image:: ../_static/doc_hierarchie.png
+
+Whenever possible, it is strongly advised not to extend the
+`vars_myProject.yaml` file, and we recommend that you use dynamic variables as
+described below. However some use cases may need to do so:
+
+* Configuring really different environments.
+* Configuration of a multi-project (see below for this specific use case).
+
+Use of dynamic variable
+.......................
+
+Variables used in the application configuration files (files `vars_xxx.yaml`)
+can be made dynamic by means of environment variable. In the main file
+`vars_myProject.yaml`, added the `interpreted` block at the bottom of the
+file.
+
+In this same file, you can change the value of a parameter by putting it in
+uppercase (example: host: HOST). This parameter must be listed in the
+interpreted parameters section::
+
+    extends: CONST_vars.yaml
+
+    vars:
+        host: HOST
+    [..]
+    interpreted:
+        environment:
+            - log_level
+            - host
+
+In the `MyProject.mk` file, add parameters you want to change and the
+`ENVIRONMENT_VARS` parameter::
```

Comment:
We don't use `ENVIRONMENT_VARS` anymore, use `export` in place :-) (as you do bellow :-) )